aws-cpp-sdk-core/source/client/AWSClient.cpp

```diff
@@ -608,6 +608,50 @@ StreamOutcome AWSClient::MakeRequestWithUnparsedResponse(const Aws::Http::URI& u
     return StreamOutcome(std::move(httpResponseOutcome));
 }
 
+#if AWS_SDK_VERSION_MAJOR > 1 || AWS_SDK_VERSION_MINOR >= 10
+StreamOutcome AWSClient::MakeRequestWithUnparsedResponse(const Aws::AmazonWebServiceRequest& request,
+                                                         const Aws::Endpoint::AWSEndpoint& endpoint,
+                                                         Http::HttpMethod method,
+                                                         const char* signerName,
+                                                         const char* signerRegionOverride,
+                                                         const char* signerServiceNameOverride) const
+{
+    Aws::Http::URI uri = endpoint.GetURL();
+    if (endpoint.GetAttributes()) {
```

Comment:
if the endpoint has no attributes can we even make a request? should we log an error and short circuit the request if it is not present?

Reply:
Frankly speaking I'm not 100% sure
It is mostly for the case when endpoint rules do not care and we fallback to the c2j-model defaults.